packages/react-native/src/storage/ReactNativeFileSystem.ts

```diff
@@ -89,4 +89,8 @@ export class ReactNativeFileSystem implements FileSystem {
     public createAttachment(path: string, name?: string | undefined): BacktraceAttachment<unknown> {
         return new BacktraceFileAttachment(this, path, name);
     }
+
+    public createWriteStream(path: string): FileWritableStream {
```

Comment:
why do we want to provide a write stream for our users? I think we don't want to manage custom write stream issues our customers might ran into

Reply:
it's not for our users, it's for us